x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/_classification_exploration.scss

```diff
@@ -1,4 +1,19 @@
-.euiFormRow.mlDataFrameAnalyticsClassification__actualLabel {
-  padding-top: $euiSize * 4;
+/* Workaround for EuiDataGrid within a Flex Layout */
+.mlDataFrameAnalyticsClassification {
+  width: calc(100% - 0px);
 }
 
+.mlDataFrameAnalyticsClassification__confusionMatrix {
+  padding: 0 5%;
+}
+
+.mlDataFrameAnalyticsClassification__actualLabel {
+  min-width: 70px;
+  padding-top: $euiSize * 4.2;
```

Comment:
why 4.2? 🤔

Reply:
It's to tweak the vertical alignment of "Actual label" against the first row of the data grid. The version before that PR used `EuiFormRow` + custom padding. I removed the form row because it was used out of context and adjusted the padding.

![image](https://user-images.githubusercontent.com/230104/81390384-9d742300-911b-11ea-8ecf-172d09abff50.png)

Reply:
have you tried to make it as `:before` pseudo-element for the header row to avoid this padding?

Reply:
Hm not sure that would be feasible to do: The header row is rendered by `EuiDataGrid` so that would mean either reusing some EUI classes and using a pseudo element would move the text to the SCSS where we cannot use translation (unless I move the style specification back into JSX?). Note this is just about adjusted an already existing padding value, it's not something this PR introduced; the main goal of this PR is to provide a fix for the broken data grid menu and fix IE. I changed the value to `$euiSize * 4 + $euiSizeXS;` to avoid having the comma in d67b062.